promptsource/promptsource.py

```diff
@@ -53,14 +53,13 @@ def reset_template_state():
 def show_jinja(t, width=WIDTH):
     wrap = textwrap.fill(t, width=width, replace_whitespace=False)
     out = highlight(wrap, DjangoLexer(), HtmlFormatter())
-    # st.write(out.replace("\n\n", "\n"), unsafe_allow_html=True)
     st.write(out, unsafe_allow_html=True)
 
 
 def show_text(t, width=WIDTH):
     wrap = [textwrap.fill(subt, width=width, replace_whitespace=False) for subt in t.split("\n")]
     wrap = "\n".join(wrap)
-    st.text(wrap)
+    st.write(wrap, unsafe_allow_html=True)
```

Comment:
the only minor thing that bothers me with `st.text` vs `st.write` is that `write` tends to mess up with the line breaks 
 and "===" sequences in the template ( i am being very picky right now, feel free to ignore)

text:
<img width="1270" alt="Capture d’écran 2021-06-05 à 10 30 32 AM" src="https://user-images.githubusercontent.com/16107619/120895079-63f53c00-c5e9-11eb-8f08-5d8f8159470f.png">

write:
<img width="1241" alt="Capture d’écran 2021-06-05 à 10 30 44 AM" src="https://user-images.githubusercontent.com/16107619/120895083-69528680-c5e9-11eb-8b28-cedea5dc4c5c.png">

Reply:
Oh yeah it must think it is markdown. I'll see if I can fix.